src/main/java/uk/gov/hmcts/reform/bsp/common/model/document/CtscContactDetails.java

```diff
@@ -17,6 +17,9 @@
     @JsonProperty("careOf")
     private String careOf;
 
+    @JsonProperty("centreName")
```

Comment:
I believe this should be called "serviceCentreName" to match the courts data structure.

Reply:
No, `centreName` is for:

```
      "serviceCentreName": "Courts and Tribunals Service Centre",
      "divorceCentre": "HMCTS Digital Divorce",  <----- this
```

`serviceCentreName` is already in `CtscContactDetails.java`. Please see line 14:

```
    @JsonProperty("serviceCentre")
    private String serviceCentre;
```

Reply:
Because so far we always used "c/o" there was a property `careOf` which is "c/o" + $divorceCentre.